types/xml-formatter/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for xml-formatter 1.1
+// Project: https://github.com/chrisbottin/xml-formatter/
+// Definitions by: Joachim Holwech <https://github.com/holwech>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace xmlFormatter;
```

Comment:
Sorry for the delay - I lost track of this one.  Is this namespace export necessary?  It's usually used to expose something in the global namespace (i.e. for webpages) and the underlying package doesn't appear to do that.

Reply:
Yeah, you are probably right. I'm not an expert on this, I can remove it if you think it shouldn't be there.

Reply:
I'd leave it out for now.  It's easy to extend typings later. :smile: